instrumented/Components/MainContainer/MainContainer.js

```diff
@@ -0,0 +1,11 @@
+import './MainContainer.css';
+import EmailNotificationForm from '../EmailNotificationForm/EmailNotificationForm';
+
+const MainContainer = () => {
+  return (
+    <main className="main-container">
+      <EmailNotificationForm />
+    </main>
+  );
+}
+export default MainContainer;
```

Comment:
I didn't see this file being used anywhere in the tests. Can you please explain what this file is for and if it's needed?

Reply:
'Explained' [here](https://github.com/mi-voz-mi-voto/mi-voz-mi-voto/pull/30#discussion_r804866273)